Doc/library/locale.rst

```diff
@@ -375,8 +375,7 @@ The :mod:`locale` module defines the following exception and functions:
    locale settings into account.
 
    .. versionchanged:: 3.7
-   Added *monetary*, if  true the conversion uses monetary thousands separator and
-   grouping strings.
+   The *monetary* keyword parameter was added.
```

Comment:
We still need the description of the monetary keyword (presumably copy and pasted from the existing format entry) in the description of the format_string function.

In fact, what we really want to do is copy most of the 'format' description into format_string, since it seems wrong somehow to refer to the docs of a deprecated from from the preferred function.